### PR TITLE
Ecdsa curve handling

### PR DESCRIPTION
**Pull request by scottbrady91**
**What issue does this PR address?**
#3531

**Does this PR introduce a breaking change?**
No more than v3 already does :)

**Please check if the PR fulfills these requirements**
- [x] The commit follows our [guidelines](https://github.com/IdentityServer/IdentityServer4/blob/master/.github/CONTRIBUTING.md)
- [x] Unit Tests for the changes have been added (for bug fixes / features)

**Other information**:
I also updated the disco doc's `id_token_signing_alg_values_supported` value to match the current signing key

### CODE REVIEW

File: src/IdentityServer4/src/Configuration/CryptoHelper.cs [hunk 1]
```diff
@@ -97,6 +98,42 @@ public static HashAlgorithm GetHashAlgorithmForSigningAlgorithm(string signingAl
...
+        }
+
+        /// <summary>
+        /// Return the matching RFC 7518 crv value for curve
+        /// </summary>
+        internal static string GetCrvValueFromCurve(ECCurve curve)
+        {
+            switch (curve.Oid.Value)
+            {
+                case "1.2.840.10045.3.1.7":
```
Comment (Randy808) [inline src/IdentityServer4/src/Configuration/CryptoHelper.cs]: Why not use the CurveOids constants defined below?

Comment (leastprivilege) [reply src/IdentityServer4/src/Configuration/CryptoHelper.cs]: @scottbrady91 comment?

Comment (scottbrady91) [reply src/IdentityServer4/src/Configuration/CryptoHelper.cs]: Because I created the constants to replace these, and then promptly forgot about them :)

Comment (scottbrady91) [reply src/IdentityServer4/src/Configuration/CryptoHelper.cs]: Updated! Thanks @Randy808

**Comment by leastprivilege** [comment]
cool! thanks!

**Pull request merged**